app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -145,7 +145,7 @@ def group_members
         .group('users.id')
         .select('users.*, COUNT(users.id) as "assignments_count"')
 
-      members = members.where("users.name like :filter OR users.username like :filter", filter: "%#{params[:filter]}%") if params[:filter]
+      members = members.where("users.name ilike ? OR users.username_lower ilike ?", "%#{params[:filter]}%", "%#{params[:filter]}%") if params[:filter]
```

Comment:
To avoid repeating the same argument twice, we can use keyword arguments like

```suggestion
      members = members.where("users.name ILIKE :pattern OR users.username_lower ILIKE :pattern", pattern: "%#{params[:filter]}%") if params[:filter]
```

Reply:
`members = members.where("users.name like :filter OR users.username like :filter", filter: "%#{params[:filter]}%") if params[:filter]` i used this as you specified above that SQL injection can be done in this😅

Reply:
🤦  I am so sorry, I read it wrong before. Your approach was 100% fine.

I thought you had done something like

```
.where("users.name LIKE #{filter}")
```

Reply:
😂😂  cool ill add a commit right now